src/task/task.json

```diff
@@ -89,13 +89,15 @@
     "metrics.inputs.adjustingFileMatchingPatterns": "Adjusting the file matching patterns input to '%s'.",
     "metrics.inputs.adjustingGrowthRate": "Adjusting the growth rate input to '%s'.",
     "metrics.inputs.adjustingTestFactor": "Adjusting the test factor input to '%s'.",
+    "metrics.inputs.adjustingChangeTitle": "Adjusting the change-title mode input to 'false'.",
```

Comment:
Please sort these elements and the ones in `task.loc.json` alphabetically. In this case:
- `metrics.inputs.adjustingChangeTitle` would be after `metrics.inputs.adjustingCodeFileExtensions`
- `metrics.inputs.disablingChangeTitle` would be after `metrics.inputs.adjustingTestFactor`

Reply:
The order is correct in `resources.resjson`, so you can just replicate the ordering from there.